File: Chapter 1/BubbleSort.py
class BubbleSort:
    def bubbleSort(self, A, n):
        for i in range(n-1, 0, -1):
            for j in range(i):
                if A[j] > A[j+1]:
                    A[j], A[j+1] = A[j+1], A[j]
        return A

File: Chapter 1/BubbleSort.cpp
class BubbleSort {
public:
    int* bubbleSort(int* A, int n) {
        // write code here
        int temp;
        for(int i=n-2; i>=1; i--){
            for(int j=0; j<=i; j++){
                if(A[j]>A[j+1]){
                    temp = A[j];
                    A[j] = A[j+1];
                    A[j+1] = temp;
                }
            }
        }
        return A;
    }
};